srcpkgs/synapse/INSTALL

```diff
@@ -0,0 +1,27 @@
+#
+# Tell user to generate configuration when they first install it
+# Taken from Arch Linux
+#
+case "${ACTION}" in
+post)
+	if [ ! -e /etc/synapse/homeserver.yaml ]; then
+		cat <<-EOF
+		==> A synapse configuration file needs to be generated before you can
+		    start synapse, and you should make sure that it's readable by the
+		    synapse user.
+
+		    cd /var/lib/synapse
+		    sudo -u synapse python -m synapse.app.homeserver \\
```

Comment:
This should be python3

Reply:
When run this pops up: `ERROR:root:Needed sortedcontainers>=1.4.4 but it was not installed
ERROR:root:Needed pymacaroons>=0.13.0 but it was not installed
Missing Requirements: "sortedcontainers>=1.4.4", "pymacaroons>=0.13.0"`